apps/examples/testcase/le_tc/filesystem/itc_fs.c

```diff
@@ -1221,7 +1221,9 @@ static void itc_libc_stdio_fseek_ftell_p(void)
 	TC_ASSERT_EQ_CLEANUP("fseek", fseek(fp, 0, SEEK_SET), OK, fclose(fp));
 	TC_ASSERT_EQ_CLEANUP("ftell", ftell(fp), (long)0, fclose(fp));
 
-	write_buf = VFS_TEST_CONTENTS_1;
+	//write_buf = VFS_TEST_CONTENTS_1;
+	strncpy(write_buf, VFS_TEST_CONTENTS_1, strlen(VFS_TEST_CONTENTS_1));
+
 	write_buf[15] = '\0';
```

Comment:
Why don't you use `strlen(VFS_XXX)` instead of constant value? You already use `strlen` above instead of 15.

Reply:
In this  case the string length is more than 16 characters(index 15),that is 18 and in between a character '\0' has been placed at [16]th location [index being 15] ,so didn't want to change this position.

Reply:
Comment has been fixed.